src/test/java/io/strimzi/kafka/bridge/http/HttpBridgeTestBase.java

```diff
@@ -85,9 +85,11 @@ static void beforeAll() {
         kafkaCluster.start();
         vertx = Vertx.vertx();
 
-        bridgeConfig = BridgeConfig.fromMap(config);
-        httpBridge = new HttpBridge(bridgeConfig);
-        httpBridge.setHealthChecker(new HealthChecker());
+        if (System.getenv("STRIMZI_USE_SYSTEM_BRIDGE") == null) {
+            bridgeConfig = BridgeConfig.fromMap(config);
```

Comment:
With external bridge we are not starting the bridge here and it is fine but what about config? Aren't tests which need info about the bridge config? Should we have the config of the external bridge?

Reply:
right now we are using it only once in the test class ConsumerGeneratedNameTest, to setup a bridge, but that's not used for external bridge.
But we can improve the tests later to handle also the configs from the external bridge as well

Reply:
do you mean that `ConsumerGeneratedNameTest` is not run with the external bridge?

Reply:
Automated tests will setup a bridge without bridgeId to run this specific test